### DISCUSSION TITLE
Ender3v2 -4.2.7 - T13 - MPC Overshoots 20*

### DISCUSSION THREAD

**Discussion by houstnwehavuhoh** [mriscoc/Ender3V2S1#591] [Q&A] [answered]
Hello,

I just installed a new 4.2.7 board on my 3v2 (with Sprite Pro Extruder) and installed the latest precompiled firmware for the T13 (the MPC version). I performed an MPC Autotune and it targeted for 200* (which was default - what I wanted anyway), however, it overshot by about 20* and never leveled to 200*. Is this correct? PID Autotune, it'd overshoot by maybe 5-7*, then level out to the desired tuning settings. With MPC, it overshot to 28*, then evened out at 220*. It then said complete. I'm about to run another autotune, but this seems a bit off?

Thanks!

EDIT: I just did a bit of digging on the Marlin page about MPC - if I'm understanding correctly, it's intentional to overshoot? Then measure the heat loss?

**Comment by mriscoc** [comment, marked as answer]
Hi, MPC is independent on temperature settings, the user must set the power of the heater, stock heater power is 40W

**Reply by houstnwehavuhoh** [reply]
Thank you!

Unrelated but, I’m having an issue where, during a print, the extruder motor is not moving. However, when I use “load filament”, it purges and the extruder motor works as intended. I’ve tried multiple slicers, cards, reflashed, etc. I’m going to attempt to use a previous firmware and see if that works, but I’m curious if there’s anything I should look into?

**Reply by mriscoc** [reply]
check if your extruder's min temperature is not higher than the current extruder temperature.

**Reply by houstnwehavuhoh** [reply]
You are a saint! I'm not sure how I changed it, but I set it higher than I was printing at. Thank you so much for the help and the amazing firmware!
